dashboard/config/routes.rb

```diff
@@ -82,12 +82,24 @@ def redirect_to_teacher_dashboard
   resources :projects, path: '/p/', only: [:index] do
     collection do
       ProjectsController::STANDALONE_PROJECTS.each do |key, value|
-        get '/' + key.to_s, to: 'levels#show', key: value, as: key.to_s
+        get '/' + key.to_s, to: 'levels#show', key: value[:name], as: key.to_s
       end
       get '/:template', to: 'projects#template'
     end
   end
 
+  resources :projects, path: '/projects/', only: [:index] do
+    collection do
+      ProjectsController::STANDALONE_PROJECTS.each do |key, value|
+        get '/' + key.to_s, to: 'projects#edit', key: key, as: key.to_s + 'project'
```

Comment:
I think I originally did this because I also had /p versions of the routes and couldn't have duplicates. You're right that it may makes sense to get rid of the + 'project' portion now.